Automattic/jetpack
# Finish replacing phpcs-excludelist.txt with .json

#### Changes proposed in this Pull Request:

A few places were missed in #18061.

#### Jetpack product discussion


None linkable.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Make a PR that introduces phpcs errors in excluded and non-excluded files. See that only the excluded ones are checked by the "PHP Code Sniffer (changes to excluded files only)" job.

#### Proposed changelog entry for your changes:


* None needed.
